src/Sql/Sql/help/Disable-AzSqlDatabaseLedgerDigestUpload.md

```diff
@@ -37,7 +37,7 @@ The Disable-AzSqlDatabaseLedgerDigestUpload cmdlet disables uploading ledger dig
 
 ### Example 1
 ```powershell
```

Comment:
Could you please follow the example template to fill examples?
\```powershell
{Code}
\```

\```output
{Output}
\```

{Description}